src/app/initialisation/run.js

```diff
@@ -4,6 +4,7 @@ var resultCore = require('fh-wfm-result/lib/client');
 var fileCore = require('fh-wfm-file/lib/client');
 var userCore = require('fh-wfm-user/lib/client');
 var $fh = require('fh-js-sdk');
+var isAuthorised = false;
```

Comment:
There's a difference between authorization and authentication. In this context we're only checking if the user is authenticated i.e. logged in.

Authorization is checking for permissions.